src/main/controllers/ContactTheTribunalController.ts

```diff
@@ -5,34 +5,37 @@ import { TranslationKeys } from '../definitions/constants';
 import applications from '../definitions/contact-applications';
 import { FormContent } from '../definitions/form';
 import { AnyRecord } from '../definitions/util-types';
+import { getFlagValue } from '../modules/featureFlag/launchDarkly';
 
 import { getPageContent } from './helpers/FormHelpers';
+import { getLanguageParam } from './helpers/RouterHelpers';
 
 /**
  * Controller for contact-the-tribunal page with a list of applications to start
  */
 export default class ContactTheTribunalController {
-  public get(req: AppRequest, res: Response): void {
+  public async get(req: AppRequest, res: Response): Promise<void> {
     const translations: AnyRecord = {
       ...req.t(TranslationKeys.CONTACT_THE_TRIBUNAL, { returnObjects: true }),
     };
-
+    const languageParam = getLanguageParam(req.url);
+    const welshEnabled = await getFlagValue('welsh-language', null);
+    const showAllSectionsText = translations.showAllSections;
```

Comment:
Do you need to create all of these consts? You should be able to access the translations in the nunjucks file

Reply:
Refactored